spanner/tests/system/test_system.py

```diff
@@ -394,6 +431,94 @@ def _unit_of_work(transaction, name):
                 COUNTERS_TABLE, COUNTERS_COLUMNS, self.ALL))
         self.assertEqual(len(rows), 2)
 
+    def test_multi_processing_map_transaction(self):
+        from multiprocessing import Pool
+        client = Client()
+        retry = RetryErrors(exceptions.ServiceUnavailable)
+        configs = list(retry(client.list_instance_configs)())
+
+        # Defend against back-end returning configs for regions we aren't
+        # actually allowed to use.
+        configs = [config for config in configs if '-us-' in config.name]
+        if len(configs) < 1:
+            raise ValueError('List instance configs failed in module set up.')
+
+        config_name = configs[0].name
+        instance = client.instance(INSTANCE_ID, config_name)
+        database_name = 'test_database' + unique_resource_id('_')
+        bursty_pool = BurstyPool()
+        database = instance.database(
+            database_name, ddl_statements=DDL_STATEMENTS, pool=bursty_pool)
+        try:
+            operation = database.create()
+            operation.result(30)  # raises on failure / timeout.
+            with database.batch() as batch:
+                batch.delete(self.TABLE, self.ALL)
+                batch.insert(self.TABLE, self.COLUMNS, self.ROW_DATA)
+            with database.snapshot() as snapshot:
+                rows = list(snapshot.read(self.TABLE, self.COLUMNS, self.ALL))
+            self._check_rows_data(rows, self.ROW_DATA)
+
+            processes = 3
+            pool = Pool(processes)
+            batches = [database_name] * processes
+            rows_list = pool.map(_process, batches)
+            pool.close()
+            pool.join()
+            self.assertEqual(len(rows_list), processes)
+            for rows in rows_list:
+                # removed a row in _process
+                self._check_rows_data(rows, self.ROW_DATA[:-1])
```

Comment:
The comment / slice doesn't make sense -- `_process` deletes all rows *before* the insert, but doesn't delete any after.

Reply:
@tseaver I removed a row from the `row_data` which populates the table in `_process` to show that I'm getting something different.  I'll address your other comments.